### PR TITLE
added testing

### PR DESCRIPTION
**Pull request by tahminaringer**

### CODE REVIEW

**Comment by tahminaringer** [comment]
- [x ] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [x ] Expected failure
     - [x ] Edge Case (if applicable/obvious)
 - [ x] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [x ] Picture of whiteboard

File: javascript/code-challenges/multiBracketValidation/multi-bracket-validation.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    if (!input.length) return false;
+    else if (element === '(' && element !== ')') return false;
+    else if (element === '{' && element !== '}') return false;
+    else if (element === '[' && element !== ']') return false;
+    else return true;
+  });
+}
+multiBracketValidation();
+
+
```
Comment (adrianhuebner) [inline javascript/code-challenges/multiBracketValidation/multi-bracket-validation.js:14]: This looks really great Tahmina, the only thing I will say if you could have the tests in a separate file that would be great!
